src/pages/vaults/VaultPositionsTable.tsx

```diff
@@ -92,17 +92,17 @@ export const VaultPositionsTable = ({ className }: { className?: string }) => {
                   >
                     {marketId === USDC_MARKET_HARDCODED ? usdcLabel : asset?.name}
                     <div tw="row gap-0.25">
-                      <$OutputSigned
-                        value={
-                          (currentLeverageMultiple ?? 0) === 0
-                            ? stringGetter({ key: STRING_KEYS.NONE })
-                            : (currentLeverageMultiple ?? 0) < 0
+                      {!!currentLeverageMultiple && (
+                        <$OutputSigned
+                          value={
+                            (currentLeverageMultiple ?? 0) < 0
```

Comment:
wait, is this conditional possible? since this won't render unless `!!currentLEverageMultiple` evaluates to true?

Reply:
sorry, what i meant is the nullish coalescing operator required*

Reply:
oh good point it's not required anymore

Reply:
welp guess it's going in the next misc fixes pr, probably tomorrow